arc-mlir/src/include/Rust/Rust.td

```diff
@@ -288,6 +289,20 @@ def Rust_RustDependencyOp : Rust_Op<"dependency", [NoSideEffect]> {
   }];
 }
 
+def Rust_RustMakeStructOp
+    : Rust_Op<"make_struct", []> {
+  let summary = "constructs a struct";
+  let description = [{
+    "rust.structt" constructs a tuple
```

Comment:
should this be `rust.structt` or `rust.struct`?

Reply:
That's a typo, will fix.